Components/RightSideBar/RightSideBar.tsx

```diff
@@ -2,30 +2,34 @@ import React from "react";
 import Image from "next/image";
 import Calendar from "./Calendar";
 import Notes from "./Notes";
+import { DarkModeToggle } from "../components";
```

Comment:
Can't we import it like this?
```suggestion
import DarkModeToggle from "../components/DarkModeToggle";
```

Reply:
Hi @lalitmee thanks for your comment, when I tried that, this is what I got 
![image](https://github.com/PiyushKalyanpy/GyanaGuru/assets/115337705/6828e97d-917c-4f45-96d6-73cda25a8ea1)
But based on your suggestion I changed it to like this and it works
![image](https://github.com/PiyushKalyanpy/GyanaGuru/assets/115337705/6d41d181-5a5a-4f6e-941c-f6a1bcba9660)

Reply:
> Do we need to add an extra class for dark mode at every place?
> 
> I think we should be able to modify the `tailwind` theme object for `dark` and `light` modes and then it should take the correct colors for whatever mode by itself.
 I am not aware of such feature. But I think in doing so we would lose scope for customization for dark mode. I think this also makes it more easy to read and understand.